### ISSUE TITLE
Reorganize project structure

### ISSUE THREAD

**Issue by patrickt** [tree-sitter/haskell-tree-sitter#128]
The structure of this project should really be
```
├── tree-sitter
├── tree-sitter-go
├── tree-sitter-haskell
├── tree-sitter-java
├── tree-sitter-json
├── tree-sitter-php
├── tree-sitter-python
├── tree-sitter-ruby
├── tree-sitter-tsx
└── tree-sitter-typescript
```

When #144 lands we should do this.

**Comment by robrix** [comment]
Strong agree :+1:

**Comment by patrickt** [comment]
Fixed with #152.
